### DISCUSSION TITLE
Why are the tags categoty NoIndex by default ?

### DISCUSSION THREAD

**Discussion by hubyhuby** [GetPublii/Publii#1503] [Help]
Each time I create a website using publii , google search is telling me that my tags are noindex... 
WHat is the use of noindex on tags pages for SEO .
My understanding is that it s more beneficial to index it , since it make internal links
![image](https://github.com/GetPublii/Publii/assets/26340942/2d902c46-32ba-417a-bb95-d9d440d5a40a)

**Comment by internettips** [comment]
Maybe the core pages that should be indexed are pages with unique, more text-rich content -- like individual blog pages, article pages, etc. Sometimes, tag pages may not have much unique text content other than a bunch of tag links / tag pictures, plus, may also have tag links / pictures,  etc., that also appear on other tag listings pages. Perhaps Google kind of 'down votes' pages that may have the same content or what it consider is 'lower value content'. So adding noindex tag helps ensure this doesn't happen. Having said that, Google seems to constantly and regularly change the way it 'values' web pages.
